lib/partial/transmuxer.js

```diff
@@ -352,6 +352,15 @@ var Transmuxer = function(options) {
     }
   };
 
+  this.setRemux = function(val) {
```

Comment:
currently this won't do anything as we don't have a `coalesceStream` for the partial transmuxer. This also means fmp4 and ts content will not work with the partial transmuxer right now.

Reply:
I believe it was intentional that there is no coalesceStream for the partial transmuxer in order to reduce the time to return data